js/main.js

```diff
@@ -3,26 +3,119 @@
 
 var $photoInput = document.getElementById('photo-input');
 var $image = document.querySelector('img');
-var $entryForm = document.querySelector('#create-form');
+var $createForm = document.querySelector('#create-form');
+var $view = document.querySelectorAll('.view');
+var $entriesLink = document.querySelector('#entries-link');
+var $newLink = document.querySelector('#new-link');
+var $noEntry = document.querySelector('#no-entry');
 
 $photoInput.addEventListener('input', handleInput);
-$entryForm.addEventListener('submit', handleSubmit);
+$createForm.addEventListener('submit', handleSubmit);
+window.addEventListener('DOMContentLoaded', handleLoad);
+$entriesLink.addEventListener('click', viewSwap);
+$newLink.addEventListener('click', viewSwap);
```

Comment:
There shouldn't be any need for these, `viewSwap` is called within `dataView` which you also have an event handler for on these elements.